script/script.js

```diff
@@ -0,0 +1,40 @@
+
+  $(document).ready(function(){
+  	
+  	var $pushtolocal = [];
+  	var $pulledfromlocal = localStorage.getItem('userinput');
```

Comment:
Because we have called JSON.stringify on our localStorage item, we can now parse it as JSON using JSON.parse
```
var $pulledfromlocal = JSON.parse(localStorage.getItem('userinput'));
```